docs/source/x_ext.rst

```diff
@@ -701,6 +726,8 @@ code bitfield of the ``mcause`` CSR. ``we`` shall be driven to 0 by the |coproce
 ``we`` is 2 bits wide when ``XLEN`` = 32 and ``X_RFW_WIDTH`` = 64, and 1 bit wide otherwise. If ``we`` is 2 bits wide, then ``we[1]`` is only allowed to be 1 if ``we[0]`` is 1 as well (i.e. for
 dual writeback).
 
+If `ecswe`` is 1, then the value in ``ecsdata`` is written to {``mstatus.xs``, ``mstatus.fs``, ``mstatus.vs``} while taking into account the WARL rules that might exist for these bitfields in the |processor|.
```

Comment:
If there is only a single write enable for all 3 extension status fields, which value should coprocessors use for a field that should not be updated? For instance, if a vector coprocessors wants to set `mstatus.vs` to `Dirty` but leave the other two fields unchanged, should it set these to `Clean` in `ecsdata`? Or must coprocessors buffer the values of other extension status fields received via the issue interface until they perform the result transaction?

Reply:
>> Or must coprocessors buffer the values of other extension status fields received via the issue interface until they perform the result transaction?

That is what I had in mind. I am also fine having ecswe 3 bits wide (1 enable per bitfield); would you prefer that?

Reply:
If you don't mind, I would be in favor of having a dedicated write enable for each field. It would avoid the overhead of buffering values that are not used in a coprocessor.

Reply:
@michael-platzer Sure, how about now?

Reply:
@Silabs-ArjanB Thank you, that looks very good to me now!